tests/contrib/asyncio/test_helpers.py

```diff
@@ -27,17 +27,17 @@ def test_ensure_future(self):
         # the wrapper should create a new Future that has the Context attached
         @asyncio.coroutine
         def future_work():
-            # the ctx is available in this task
-            ctx = self.tracer.get_call_context()
-            assert 1 == len(ctx._trace)
-            assert "coroutine" == ctx._trace[0].name
-            return ctx._trace[0].name
+            return self.tracer.trace("child")
 
-        self.tracer.trace("coroutine")
+        s1 = self.tracer.trace("coroutine")
         # schedule future work and wait for a result
         delayed_task = helpers.ensure_future(future_work(), tracer=self.tracer)
-        result = yield from asyncio.wait_for(delayed_task, timeout=1)
-        assert "coroutine" == result
+        s2 = yield from asyncio.wait_for(delayed_task, timeout=1)
+        s2.finish()
+        s1.finish()
+        assert s1.parent_id is None
+        assert s2.parent_id == s1.span_id
+        assert s1.trace_id == s2.trace_id
```

Comment:
do you still want to check the name of the s1 trace?

Reply:
not a biggie, but I can add it back since the test previously had it